code/acting/src/acting/pure_pursuit_controller.py

```diff
@@ -59,7 +66,7 @@ def __init__(self):
         self.__position: (float, float) = None  # x, y
         self.__last_pos: (float, float) = None
         self.__path: Path = None
-        self.__heading: float = None
+        self.__heading: float = 0.0
```

Comment:
It might not be useful to publish steering angles if actual heading hasn't been received yet

Reply:
fixed in 3edfc76